README.md

```diff
@@ -58,6 +58,28 @@ Additional env vars that affect the generation of the docs:
   ```bash
   SKIP_DEPRECATED=true MAX_FILES=10 CONTENT_REPO_DIR=~/dev/demisto/content npm run reference-docs
   ```
+  
+  
+## Generating Marketplace Docs
+Marketplace docs are generated from the content google cloud bucket. If you are working on general site docs which are not related to marketplace, you don't need to run this step to preview the marketplace tab.
+
+To generate the docs we use [pipenv](https://github.com/pypa/pipenv). Make sure to install pipenv by running: `pip3 install pipenv`.
+
+When working locally you can generate the reference docs by running:
+```
+npm run marketplace-docs
+```
+This task will download the packs information from the google cloud bucket and generate the packs docs for the marketplace tab. The generated docs are ignored by `.gitignore` and shouldn't be checked in as they are generated during the build.
+
+Additional env vars that affect the generation of the docs:
+* `FILE_REGEX`: specify a regex to select which docs to generate docs for. For example: 
```

Comment:
is `FILE_REGEX` relevant?

Reply:
Thought about implementing it per pack, I will leave it tp phase two and remove for now